src/dataflow/source/kinesis.rs

```diff
@@ -171,6 +158,18 @@ where
                     output.session(&cap).give((data, None))
                 }
 
+                if let Some(millis) = get_records_output.millis_behind_latest {
```

Comment:
Definitely don't go back through CI for this, but I think `if let Some(0) = get_records_output.millis_behind_latest` would work here!